function_5_3.py

```diff
@@ -0,0 +1,37 @@
+"""Write a program that asks the user how many Fibonacci numbers to generate and then generates them.
+Take this opportunity to think about how you can use functions.
+Make sure to ask the user to enter the number of numbers in the sequence to generate.
+(Hint: The Fibonacci sequence is a sequence of numbers where the next number in the sequence is the
+sum of the previous two numbers in the sequence. The sequence looks like this: 1, 1, 2, 3, 5, 8, 13, …)
+"""
+
+
+# 0,1,1,2,3,5,8,13,21,34,55,
+# 0, 1, 1+0, 1+1, 2+1, 2+3, 3+5,
+import sys
+
+from pip._vendor.distlib.compat import raw_input
+
+
+def fibo(x):
+    fib = [0]
+    a=0
+    b=1
+    for i in range(1, x, 1):
+        c = a + b
+        a=b
+        b=c
+        fib.append(c)
+    return fib
```

Comment:
Fibonacci sequence generated by the program is not correct is should be:
1,1,2,3,5,...